pyproject.toml

```diff
@@ -29,4 +31,23 @@ addopts = "-ra --cov"
 testpaths = [
     "tests"
 ]
-pythonpath = ['.']
+pythonpath = ['src']
```

Comment:
This snuck in but I was running into testing issues since the folder structure changed

Reply:
Makes sense, yeah